public/ipython/notebook/js/outputarea.js

```diff
@@ -982,6 +986,24 @@ define([
         }
     };
 
+    OutputArea.prototype.convertSvg = function() {
+        for (var i = 0; i < this.outputs.length; i++) {
+            var o = this.outputs[i];
+            if (o.output_type == 'execute_result') {
+                if (o.data["text/html"]) {
+                    var h = o.data["text/html"];
+                    var svg = this.element.find("svg");
+                    if (svg.length) {
```

Comment:
don't get it, what do you mean?
This dummy check is to see if there is one.

But maybe you're worried about the size encoded stuff (+ the SVG itself...)